pkg/server/handler.go

```diff
@@ -370,12 +372,33 @@ func newActiveListener(listener types.Listener, logger log.Logger, accessLoggers
 	al.statsNamespace = types.ListenerStatsPrefix + strconv.Itoa(listenPort)
 	al.stats = newListenerStats(al.statsNamespace)
 
+	mgr, err := tls.NewTLSServerContextManager(lc, listener, logger)
+	if err != nil {
+		logger.Fatalf("create tls context manager failed, %v", err)
```

Comment:
Do not use fatal log here, newActiveListener may be called on dynamic resource case